### ISSUE TITLE
Add keyboard navigation for results

### ISSUE THREAD

**Issue by fouad** [fouad/chrome-tablight#3]
Add other cases at [app/scripts/contentscript.js#L65-L78](https://github.com/fouad/chrome-tablight/blob/master/app/scripts/contentscript.js#L65-L78) for up/down arrow to select result cards

**Comment by tekumara** [comment]
Agreed - would make it much more useful (Tab Ahead has this feature)
